### PR TITLE
Fix #702: Add Profile Update Layout

### LINKED ISSUE #702: Lowfi - Add profile updated
Mock Link: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/77b5658b-cd35-4eda-b971-f1f4fd0181de/PC-MP-Add-Profile-Empty-7

The current implementation in develop branch for `Add Profile` is different as compared to the new mocks.

Please update this. Also, write test cases for the same.

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@rt4914 do I need to change the logic for profile creation?

**Comment by aggarwalpulkit596** [comment]
ezgif com-video-to-gif

File: app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt [hunk 1]
```diff
@@ -81,6 +82,15 @@ class AddProfileActivityTest {
     }
   }
 
+  @Test
+  fun testAddProfileActivity_createPin_checkOpensProfileActivity() {
+    ActivityScenario.launch(AddProfileActivity::class.java).use {
+      onView(allOf(withId(R.id.checkbox_pin))).perform(
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt]: Single line:
`onView(allOf(withId(R.id.checkbox_pin))).perform(click())`

File: app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt [hunk 2]
```diff
@@ -81,6 +82,15 @@ class AddProfileActivityTest {
...
+        click()
+      )
+      onView(withId(R.id.input_pin)).check(matches(isDisplayed()))   }
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt]: `}` should go to next line.

File: app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt [hunk 3]
```diff
@@ -81,6 +82,15 @@ class AddProfileActivityTest {
...
+  }
+
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt:99]: Add one more test case, which makes sure that by default input_pin is not visible.

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, thanks a lot. Just add one test case.

**Comment by rt4914** [comment]
@aggarwalpulkit596 The branch name that you have kept is not correct.

We should write it in some more explaining way line `update-add-profile` or something else.

**Pull request closed without merging**